### ISSUE TITLE
How to view the evaluation results of the interpolation

### ISSUE THREAD

**Issue by DustSettled** [vLAR-group/NVFi#7]
Hello, author. I noticed that in the document, you only specified the evaluation process of the Extrapolation experiment, but did not clearly explain how the interpolation is evaluated. Could you please tell me about it? Thank you.

**Comment by Jinxi-Li** [comment]
Yes, let me take an example:
A scene includes 20 cameras, and we take 17 views as training views and 3 views as novel views, and we set $t=0.75$ as the maximum observed time (you can also choose the largest time in training set). Then,

**part1:** $t=0$ to $t=0.75$, training views: _training_
**part2:** $t=0$ to $t=0.75$, novel views: _evaluating novel-view interpolation_
**part3:** $t=0.75$ to $t=1$, all (training + novel) views: _evaluating future extrapolation_

Since currently, our dataset format includes both **part2** and **part3** in test set, so they are rendered and evaluated all together. You can manually separate them apart by time. For example, you can filter [evaluating time](https://github.com/vLAR-group/NVFi/blob/ff63e1b55a5612656b16871034345e4f4b2896bd/train_nvfi.py#L434) when rendering and evaluating.

**Comment by DustSettled** [comment]
Thank you for your answers. However, when I am reproducing this work, I find that there is a significant discrepancy between my experimental results and those in your paper regarding the four scenes of the InDoorSeg Dataset (Dynamic Indoor Scene Dataset). I have strictly followed the documentation you provided for the reproduction. The PSNR values of my reproduced results for all four scenes are less than 20, while those in your paper are basically around 30. Could you please help me solve this problem? Thank you very much.

**Comment by Jinxi-Li** [comment]
Can you briefly describe how the qualitative results look like? Or you can send some qualitative results to me via email so that I can have a better sense.

**Comment by DustSettled** [comment]
Okay, let me describe my reproduction results: chessboard: 'psnr': 18.89; darkroom: 'psnr': 19.91; dining: 'psnr': 16.68; factory: 'psnr': 18.30. Later, I modified the configuration file (.yam) and ran another round of experiments, but the PSNR was generally only around 20.

**Comment by Jinxi-Li** [comment]
Only by looking at the numbers I got not idea. Have you taken a look at the qualitative results? Do they look reasonable or not?

**Comment by DustSettled** [comment]
The results are extremely blurry, with severe artifacts and low resolution. These are some of my experimental findings.：
<img width="200" height="200" alt="Image" src="https://github.com/user-attachments/assets/aa3647d1-6490-4af3-9b4a-2b90ef0c4902" />
<img width="400" height="400" alt="Image" src="https://github.com/user-attachments/assets/443ed2ac-bab5-4870-a09e-5776c07129c0" />
<img width="200" height="200" alt="Image" src="https://github.com/user-attachments/assets/fba071be-e4c7-43ed-b718-963dc88f673e" />
<img width="200" height="200" alt="Image" src="https://github.com/user-attachments/assets/e7e6160d-6505-4ffe-ae86-595f3948b97d" />

**Comment by Jinxi-Li** [comment]
I'll have a test later and respond to you. I haven't met this problem before.

**Comment by DustSettled** [comment]
Dear author, if you find out the reason, please let me know. I am in urgent need of replicating this part of the experiment.

**Comment by Jinxi-Li** [comment]
I have tested the codes, and I found training with higher functorch version will influence the performance. But I still cannot replicate your case. May I know your exact training environment?

**Comment by DustSettled** [comment]
As for the training environment I use, the operating system is Ubuntu 22.04.2 LTS.

<img width="757" height="454" alt="Image" src="https://github.com/user-attachments/assets/3eb6f4e3-ce1c-4699-b257-6bf9f46e1369" />
<img width="1002" height="479" alt="Image" src="https://github.com/user-attachments/assets/d3876c0b-37c9-4cf7-a226-e6cad876fce9" />

**Comment by DustSettled** [comment]
Hello author, I have been unable to successfully install Open3D due to installation issues. However, your document states that it only needs to be installed when using the segmentation application. I wonder if this is the reason (for the problem I'm encountering)?

**Comment by Jinxi-Li** [comment]
I have replicated yuor results with the same environment setting. So it seems this is a problem caused by CUDA, torch, functorch versions, but I don't know which of them. So you are recommended to use the same version as suggested. Or if you need it, I can release the checkpoint for you tomorrow.

**Comment by DustSettled** [comment]
Okay, thank you so much. Can this problem be solved? If it can't be resolved, I'll reinstall the versions of CUDA, torch, and functorch. Thank you very much for your explanation.

**Comment by Jinxi-Li** [comment]
SInce I am too busy working on another project now, I don't have enough time to diagnose which part is influenced  and how currently. Maybe you can try to reinstall the environment. 

BTW, let me recommend you to use our newest work, based on Gaussian splatting, finishing the same tasks as NVFi, called [FreeGave](https://github.com/vLAR-group/FreeGave)! That one is much more powerful, and I guess the reliance of that one could be much newer. More compatible for your A100 GPU.

**Comment by DustSettled** [comment]
Okay, thank you.

**Comment by DustSettled** [comment]
Dear Author,

After my efforts, I have resolved most of the issues. However, the same problem still persists with the "factory" scene in the InDoorSeg dataset. I suspect there might be an issue with the dataset itself. I downloaded the dataset from the Google Drive link provided in your documentation.
